README.md

```diff
@@ -1,2 +1,40 @@
-# Vivian's research project
-* requires structured_random_features repo. follow instructions to set up here: https://github.com/glomerulus-lab/structured-random-features
+# Learning and Aligning Structured Random Feature Networks
+This is the repo for the Learning and Aligning Structured Random Feature Networks paper, accepted at the ICLR Re-Align 2024 workshop. The important code relevant to the paper is located in the `FactConv` directory.
```

Comment:
"The important code relevant" could be better phrased. "The most important used in the paper"?